docs/_sass/_main.scss

```diff
@@ -144,6 +144,21 @@ p {
     padding-bottom: 20px;
 }
 
+code {
+    font-family: 'Expensify Mono', 'SFMono-Regular', monospace, Arial, sans-serif;
+    border-width: 1px;
+    border-style: solid;
+    border-radius: 5px;
+    border-color: var(--color-borders);
+    background-color: var(--color-highlightBG);
+    line-height: 20px;
+    padding-left: 5px;
+    padding-right: 5px;
+    font-size: 13px;
```

Comment:
Can you check and see if we have a variable for this size? Like font-size-label or something?

Reply:
Cool, thanks for confirming.